x-pack/plugins/security_solution/common/types/timeline/server.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { RowRendererId, Sort, TimelineStatus } from '@kbn/timelines-plugin/common/types';
+import type { FavoriteTimelineResult } from '../../../public/timelines/components/open_timeline/types';
+
+import type { Maybe } from '../../search_strategy';
+
+import type {
+  ColumnHeaderResult,
+  DataProviderResult,
+  DateRangePickerResult,
+  EqlOptionsResult,
+  FilterTimelineResult,
+  SerializedFilterQueryResult,
+  TimelineType,
+} from '..';
+import type { NoteResult } from './note';
+import type { PinnedEvent } from './pinned_event';
+
+export interface TimelineServerResponse {
```

Comment:
Thanks for separating this out

Reply:
Haha, yeah. I think the type will eventually live in that file but for now it's not used.

Reply:
It's all in `api.ts` now (https://github.com/elastic/kibana/pull/158566/commits/16c7a5fa83d26288cd508bd3cf6314bb8f94b717 and following commits)